### PR TITLE
doc: Update README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
- Updated README file by formatting some text.
- Added Contributors section.
- Made the info concise to look.

## Screenshots

See the changes [HERE](https://github.com/utkarsh006/openfoodfacts-events/tree/ReadmeUpdate)

### CODE REVIEW

**Comment by utkarsh006** [comment]
@VaiTon Kindly review.

File: README.md [hunk 1]
```diff
@@ -7,13 +7,18 @@ system.
 
 To run the API locally, run `make dev`. This assumes you have `Makefile`, `Docker` and `docker-compose` installed on your machine.
 
+<br>
+
+![Github Repo Size](https://img.shields.io/github/repo-size/openfoodfacts/openfoodfacts-events?style=for-the-badge&color=aqua)
+
+
```
Comment (VaiTon) [inline README.md:14]: Is this really something we need? The repo is not that big

File: README.md [hunk 2]
```diff
@@ -7,13 +7,18 @@ system.
...
 ## API Documentation
 
-The API documentation is available at https://events.openfoodfacts.net/docs.
+The API documentation is available [HERE](https://events.openfoodfacts.net/docs)
 
 ## Examples
 
-### cURL
+<details><summary><h2> CURL </h2></summary>
```
Comment (VaiTon) [inline README.md:21]:
```suggestion
<details><summary><h2> curl </h2></summary>
```

**Review by VaiTon — COMMENTED** [review]
Hey! Thank you for your time!

Just a little comment about the repo size and "curl" should be spelled like that IMO (at least that is what the official website goes for)

File: README.md (hunk 1 above)
Comment (utkarsh006) [reply README.md:14]: Yes I think it will be better for the contributors to know this fact.

**Pull request closed without merging**